egui/src/widgets/button.rs

```diff
@@ -239,29 +239,40 @@ impl<'a> Widget for Checkbox<'a> {
         let icon_width = spacing.icon_width;
         let icon_spacing = ui.spacing().icon_spacing;
         let button_padding = spacing.button_padding;
-        let total_extra = button_padding + vec2(icon_width + icon_spacing, 0.0) + button_padding;
+        let mut total_extra = button_padding;
+        if !text.is_empty() {
+            total_extra += vec2(icon_width + icon_spacing, 0.0) + button_padding;
+        }
 
         let wrap_width = ui.available_width() - total_extra.x;
```

Comment:
You only need to calculate `wrap_width` if there is text to wrap, so you only need to calculate `total_extra` if there is text.

Reply:
Hi, thank you for the feedback - I still would like to see this merged.
So, I have tried to implement it.
If this is OK, I would copy it to radio-button.